dev_config/src_acctsc_rollups/package.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
+        <members>Account.AccConSC_Last_SC_AMt__c</members>
+        <members>Account.AccConSC_Total_SCs__c</members>
+        <members>Account.AccConSC__c</members>
+        <members>Account.Largest_SC_Amount__c</members>
+        <members>Account.SC_Avg_SC_Amt__c</members>
+        <members>Account.SC_First_SC_Amount__c</members>
+        <members>Account.SC_Largest_SC_Date__c</members>
+        <members>Account.SC_Last_SC_Date__c</members>
+        <members>Account.SC_SC_Best_Year_Total__c</members>
+        <members>Account.SC_SC_Best_Year__c</members>
+        <members>Account.SC_Smallest_SC_Amt__c</members>
+        <members>Account.SC_Total_Amt_SCs_2_Yrs_Ago__c</members>
+        <members>Account.SC_Total__c</members>
+        <members>Account.SCs_Last_Year__c</members>
+        <name>CustomField</name>
+    </types>
+    <types>
+        <members>Rollup.AcctAccConSCAvgSCAmtb04b7da5</members>
+        <members>Rollup.AcctAccConSCLastSCAmta6748a70</members>
+        <members>Rollup.AcctAccConSCTotalSCs5c0ce72c</members>
+        <members>Rollup.AcctSCAvgSCAmt07d37166</members>
+        <members>Rollup.AcctSCLargestSCAmount5f162240</members>
+        <members>Rollup.AcctSCLargestSCDateeaa0a093</members>
+        <members>Rollup.AcctSCLastSCDateb2b4b787</members>
+        <members>Rollup.AcctSCNumSCLastYear7d2ae551</members>
+        <members>Rollup.AcctSCSCBestYear52a61eb6</members>
+        <members>Rollup.AcctSCSCBestYearTotal33464ad2</members>
+        <members>Rollup.AcctSCSmallestSCAmt526efbf3</members>
+        <members>Rollup.AcctSCTotalAmtSCs2YrsAgo56adfbf4</members>
+        <members>Rollup.AcctSCTotalAmtSCsd2d11dff</members>
+        <members>Rollup.ConFirstSoftCreditAmounta60c284d</members>
+        <name>CustomMetadata</name>
+    </types>
+    <version>39.0</version>
```

Comment:
I suggest using API v43 to match the current api version. I have a separate PR bringing the rest of the code base up to this version.

Reply:
Sure; I updated it.